enterprise/server/raft/replica/replica.go

```diff
@@ -756,6 +756,8 @@ func (sm *Replica) scan(db ReplicaReader, req *rfpb.ScanRequest) (*rfpb.ScanResp
 		t = iter.SeekLT(req.GetLeft())
 	case rfpb.ScanRequest_SEEKGE_SCAN_TYPE:
 		t = iter.SeekGE(req.GetLeft())
+	case rfpb.ScanRequest_SEEKGT_SCAN_TYPE:
+		t = iter.SeekGE(keys.Key(req.GetLeft()).Next())
```

Comment:
I think this is wrong. You only want to get the next key if the key the iterator points to is equal to the target key.

Reply:
PTAL